### ISSUE TITLE
[4.0] "Highlight Links" partial not shown on sidebar icons

### ISSUE THREAD

**Issue by ghost** [joomla/joomla-cms#34410]
### What needs to be fixed

 Enable in user profile the accessibility setting `Highlight Links`. Some icons on the sidebar look as having no underline (articles) or only a part underlined (components):

![120783841-ba9c4080-c52b-11eb-964f-c90a27bd1d9c](https://user-images.githubusercontent.com/78906980/120879217-2a5d0a80-c5c2-11eb-82f5-7b88cb8bae77.png)

### Why this should be fixed

All icons on sidebar should links in same length.

### How would you fix it
Don't know.

### Side Effects expected
I hope not.

**Comment by chmst** [comment]
Thank for reporting. Underlined icons look completely wrong. It is a css thing but I am not sure if icons should be underlined at all.

**Comment by brianteeman** [comment]
Well they are links so I guess they should be when highlihgt links is on.

The problem you have observed is that the icon is rendered on top of the underline. So you cant see it on the content icon and for the exxtensions plugin you can only see a tiny part of it on the right.

Not sure if its worth the time trying to fix it unless its very simple

**Comment by Aditi-Singh16** [comment]
Hello, I am new to Opensource, I would like to fix this issue, where would I find the file where changes have to be made?

**Comment by chmst** [comment]
Thank you for your interest in contributing!  Your question ist THE question .. where can we fix this?
